### PR TITLE
add spigot tests

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

File: acd-screen-share-test.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* global describe, it, before, afterEach, after */
+import { PureCloudWebrtcSdk } from '../../dist/purecloud-webrtc-sdk';
+import { testUtils, callUtils } from 'purecloud-spigot';
```
Comment (djhouseknecht) [inline acd-screen-share-test.js]: NIT: Maybe we should name it `genesysclout-spigot` before we get too far 😬

Comment (jensengar) [reply acd-screen-share-test.js]: that's a good idea. I always forget about the name change.

File: spigot-tests/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "chai": "^4.2.0",
+    "purecloud-spigot": "^1.0.0",
+    "purecloud-streaming-client": "^11.0.2",
+    "uuid": "^7.0.1"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "mocha",
+    "start": "SINGLE_RUN=true node index.js",
+    "watch": "node index.js"
```
Comment (djhouseknecht) [inline spigot-tests/package.json]: Does this `watch` start the tests in `watch` mode?

Comment (jensengar) [reply spigot-tests/package.json]: yes. I forgot to change this to match the standard format you suggested. I'll do that

**Pull request merged**